docker/minio/secret_key.txt

```diff
@@ -1 +0,0 @@
-SLIMEVRSECRET
```

Comment:
this directory / file does still not exist: https://github.com/SlimeVR/SlimeVR-Firmware-WebBuilder/tree/llelievr/new-fw-tool-api
maybe because it is in [.gitignore](https://github.com/SlimeVR/SlimeVR-Firmware-WebBuilder/blob/llelievr/new-fw-tool-api/.gitignore#L38C1-L38C13)